cli/azd/cmd/login.go

```diff
@@ -294,8 +284,7 @@ func (la *loginAction) login(ctx context.Context) error {
 		return nil
 	}
 
-	useDeviceCode := la.flags.useDeviceCode || os.Getenv(CodespacesEnvVarName) == "true" ||
-		os.Getenv(RemoteContainersEnvVarName) == "true"
+	useDeviceCode := la.flags.useDeviceCode
```

Comment:
super minor nit (not work restarting the run over, we can just clean it up next time) but I would have just updated L289 to use `la.flags.useDeviceCode` instead of creating the local since we don't need it anymore.

Reply:
agree. Updated